### DISCUSSION TITLE
Extension isnt working for me

### DISCUSSION THREAD

**Discussion by maevyyy** [uBlockOrigin/uBOL-home#172] [Q&A]
hi, i installed this extenension 2 days ago at first it worked, but now it doesnt(youtube). i have is set to Complete mode. i have a chromium based browser.

**Comment by gorhill** [comment]
Read and follow all the steps carefully: https://github.com/uBlockOrigin/uAssets/issues/20586

If after following all instructions to the letter it still doesn't work, not much we can do with uBOL, you will have to find another blocker.
